doc/source/ray-air/examples/torch_detection.ipynb

```diff
@@ -0,0 +1,1081 @@
+    "In this example, you'll train an object detection model on\n",
+    "[Pascal VOC](http://host.robots.ox.ac.uk/pascal/VOC/), a canonical object detection\n",
+    "dataset. The dataset contains 11,530 images across 20 different classes:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "a99ec77e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "CLASS_TO_LABEL = {\n",
+    "    \"background\": 0,\n",
+    "    \"aeroplane\": 1,\n",
+    "    \"bicycle\": 2,\n",
+    "    \"bird\": 3,\n",
+    "    \"boat\": 4,\n",
+    "    \"bottle\": 5,\n",
+    "    \"bus\": 6,\n",
+    "    \"car\": 7,\n",
+    "    \"cat\": 8,\n",
+    "    \"chair\": 9,\n",
+    "    \"cow\": 10,\n",
+    "    \"diningtable\": 11,\n",
+    "    \"dog\": 12,\n",
+    "    \"horse\": 13,\n",
+    "    \"motorbike\": 14,\n",
+    "    \"person\": 15,\n",
+    "    \"pottedplant\": 16,\n",
+    "    \"sheep\": 17,\n",
+    "    \"sofa\": 18,\n",
+    "    \"train\": 19,\n",
+    "    \"tvmonitor\": 20,\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3b143018",
+   "metadata": {},
+   "source": [
+    "### Download Pascal VOC"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4d1f120e",
+   "metadata": {},
+   "source": [
+    "First, download the 2GB of raw data:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "bf966cc5",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current\n",
+      "                                 Dload  Upload   Total   Spent    Left  Speed\n",
+      "100 1907M  100 1907M    0     0  9786k      0  0:03:19  0:03:19 --:--:--  9.8M:19  0:03:08  0:00:11 9947k\n"
+     ]
+    }
+   ],
+   "source": [
+    "!curl -OJ http://host.robots.ox.ac.uk/pascal/VOC/voc2012/VOCtrainval_11-May-2012.tar"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "449fa021",
+   "metadata": {},
+   "source": [
+    "Then, untar the raw data to create the `VOCdevkit/VOC2012` folder:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "7ca845d5",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "!tar -xf VOCtrainval_11-May-2012.tar"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "692d18c5",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Annotations  ImageSets\tJPEGImages  SegmentationClass  SegmentationObject\n"
+     ]
+    }
+   ],
+   "source": [
+    "!ls VOCdevkit/VOC2012"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "5b66c4cc",
+   "metadata": {},
+   "source": [
+    "### Define a custom datasource"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "934770d7",
+   "metadata": {},
+   "source": [
+    "Each image has an annotation:"
```

Comment:
the text in this section doesn't correspond to the following code, which opens the image itself but not the annotation. Maybe we could replace it with something like, "After downloading the dataset, we can view each image:" and combine with the next text to say "Each image has an annotation, which describes the objects in an image:"

Reply:
Ah, good catch. Must've messed this up while revising.